firefox/guide/firefox/firefox_preferences-development_tools/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+title: 'Firefox Development Tools Must Be Disabled.'
```

Comment:
Let's make this: `Disable Firefox Development Tools`

Reply:
Edited the title accordingly